core-to-plc/src/Language/Plutus/CoreToPLC.hs

```diff
@@ -567,7 +773,7 @@ convOrdMethod name
     | GHC.getOccString name == "<" = pure $ mkIntRel PLC.LessThanInteger
     | otherwise = throwSd UnsupportedError $ "Ord method:" GHC.<+> GHC.ppr name
 
-convNumMethod :: (Converting m) => GHC.Name -> m PLCExpr
+convNumMethod :: (Converting m) => GHC.Name -> m PLCTerm
 convNumMethod name
     -- only this one has a name defined in the lib??
```

Comment:
It has just occurred to me that `-` is a rather unique thing: it's treated sometimes like a prefix operator (the only in Haskell?) and sometimes like an infix operator, so no surprise it has a name when others don't. Not sure whether this was said before.